portal-api/src/v1/users/sign-in-link.service.js

```diff
@@ -30,14 +30,28 @@ class SignInLinkService {
 
   async isValidSignInToken({ userId, signInToken }) {
     const user = await this.#userRepository.findById(userId);
+
     if (!user.hash || !user.salt) {
       throw new Error('User does not have a valid sign in token.');
```

Comment:
Just wondering what this is checking? These are the `hash` and `salt` on the top-level `user` object - to me, this seems to have more to do with whether the user has set a password rather than the validity of the sign in token.

Reply:
@AlexBramhill has fixed this in https://github.com/UK-Export-Finance/dtfs2/pull/2300/